index.test-d.ts

```diff
@@ -62,6 +63,8 @@ conf.set('unicorn', false);
 
 expectType<string>(conf.get('foo'));
 expectType<string>(conf.get('foo', 'bar'));
+conf.reset('foo');
+expectType<string>(conf.get('foo'));
```

Comment:
This assertion is meaningless, as it's just a type assertion and we already checked `.get`.